inst/validation/specs.R

```diff
@@ -0,0 +1,7 @@
+# Use a list to declare the specs
```

Comment:
how will this list be consumed from external repos?

Reply:
It is not consumed by other packages, each package will have its own copy of the qc scripts. We can modify this in the future so packages consume qc scripts stored in dv.templates

Reply:
> We can modify this in the future so packages consume qc scripts stored in dv.templates

not sure I get then why are we hosting the /inst/validation + /scripts here then since all the target repos should have them....

Reply:
We are just testing that the qc report part runs fine and store the basic structure of a package, the same way we store tests and the R directory.